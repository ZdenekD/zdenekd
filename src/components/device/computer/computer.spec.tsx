import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '@/__test__/utils/helpers';
import Computer from '.';

describe('Computer device', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Computer />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-device-computer');

        expect(component.exists()).toBe(true);
    });
});
