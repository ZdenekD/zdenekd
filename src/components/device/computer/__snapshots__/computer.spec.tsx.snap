// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Computer device match snapshot 1`] = `
<div class="wrapper"
     data-test="component-device-computer"
>
  <div class="computer">
    <div class="display">
      <div class="header">
        <i class="camera">
        </i>
      </div>
      <div class="content">
      </div>
      <div class="footer">
      </div>
    </div>
    <div class="keyboard">
    </div>
    <div class="backside">
    </div>
  </div>
</div>
`;
